src/controllers/controller.polarArea.js

```diff
@@ -211,12 +225,30 @@ module.exports = function(Chart) {
 			return count;
 		},
 
-		calculateCircumference: function(value) {
+		_computeAngle: function(index) {
```

Comment:
We started to be even more explicit about private stuff by adding the following comment above:

```javascript
/**
 * @private
 */
_computeAngle: function(index) {
```